viewer/packages/tools/src/Measurement/types.ts

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+export type MeasurementLineOptions = {
+  lineWidth?: number;
+  color?: number;
+};
+
+export type MeasurementUnitUpdateDelegate = () => MeasurementUnits;
+
+export type MeasurementOptions = {
+  unitsUpdateCallback?: MeasurementUnitUpdateDelegate;
```

Comment:
Does it make sense that this is optional? I feel like an empty object doesn't make sense as an object.